report/src/main/kotlin/com/hartwig/actin/report/pdf/chapters/ClinicalDetailsChapter.kt

```diff
@@ -18,18 +26,47 @@ class ClinicalDetailsChapter(private val report: Report, override val include: B
         return PageSize.A4
     }
 
+    override fun include(): Boolean {
+        return configuration.clinicalChapterType != ClinicalChapterType.NONE
+    }
+
     override fun render(document: Document) {
         addChapterTitle(document)
         addClinicalDetails(document)
     }
 
     private fun addClinicalDetails(document: Document) {
         val table = Tables.createSingleColWithWidth(contentWidth())
+        TableGeneratorFunctions.addGenerators(createClinicalDetailGenerators(), table, overrideTitleFormatToSubtitle = false)
+        document.add(table)
+    }
+
+    fun createClinicalDetailGenerators(): List<TableGenerator> {
```

Comment:
breaks encapsulation for sake of a test. could be tempting for some non-test code to call this function. I would try to assert against the Document if I want to ensure the contents are included.